pylint_plugins/fixtures/api_models.py

```diff
@@ -0,0 +1,159 @@
+# Copyright 2022 The StackStorm Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import abc
+
+DEFAULT_PACK_NAME = "default"
+
+
+NotificationSubSchemaAPI = {
+    "type": "object",
+    "properties": {
+        "message": {"type": "string", "description": "Message to use for notification"},
+        "data": {
+            "type": "object",
+            "description": "Data to be sent as part of notification",
+        },
+        "routes": {
+            "type": "array",
+            "description": "Channels to post notifications to.",
+        },
+        "channels": {  # Deprecated. Only here for backward compatibility.
+            "type": "array",
+            "description": "Channels to post notifications to.",
+        },
+    },
+    "additionalProperties": False,
+}
+
+
+def get_schema(**kwargs):
+    return {}
+
+
+# copied from st2common.models.api.base
```

Comment:
If we change the base classes in st2 do we need to change these?
If so, can we add a comment on the st2 files themselves.
But I am assuming we don't need to.

Reply:
Yeah a comment would probably make sense.

It's ok if they drift (the precise options don't matter for the test). But if we do any more major refactoring, then we need to also update this so our test works.

Reply:
I added comments about this.